lib/rspec/rails/matchers/action_cable/have_broadcasted_to.rb

```diff
@@ -140,9 +144,14 @@ def set_expected_number(relativity, count)
               end
           end
 
-          def base_message
+          def base_description
             "#{message_expectation_modifier} #{@expected_number} messages to #{stream}".tap do |msg|
               msg << " with #{data_description(@data)}" unless @data.nil?
+            end
+          end
+
+          def base_message
+            base_description.tap do |msg|
               msg << ", but broadcast #{@matching_msgs_count}"
             end
```

Comment:
This minorly improve allocations 
```suggestion
            "#{base_description}, but broadcast #{@matching_msgs_count}"
```